src/main/java/duck/operation/Command.java

```diff
@@ -0,0 +1,148 @@
+package duck.operation;
+
+import duck.task.Deadline;
+import duck.task.Event;
+import duck.task.TaskList;
+import duck.task.Todo;
+
+import java.io.IOException;
+import java.time.format.DateTimeParseException;
+
+public class Command {
+    private String command;
+    private String description;
+
+    /**
+     * Initialize the Command object
```

Comment:
Would it be better to leave an empty line between the description and parameter section of the Javadocs (increases readability too)? The description should also begin with 'Initialises'. Just to be in line with the module's conventions.